Cargo.toml

```diff
@@ -256,7 +256,7 @@ datafusion-common-runtime = { git = "https://github.com/spiceai/datafusion.git",
 datafusion-sql = { git = "https://github.com/spiceai/datafusion.git", rev = "5897255853b7f2b60aa29fe640ae238ec6288a7c" } # spiceai/spiceai-49
 
 datafusion-federation = { git = "https://github.com/spiceai/datafusion-federation.git", rev = "5ad2f52b9bafc6eaa50851f2e1fcf0585fb5184d" }                      # spiceai-49
-datafusion-table-providers = { git = "https://github.com/datafusion-contrib/datafusion-table-providers.git", rev = "e751fb367eb29f299683ff5125f8da2f654e0479" } # spiceai
+datafusion-table-providers = { git = "https://github.com/datafusion-contrib/datafusion-table-providers.git", rev = "1f5d824a6b476c9dab5e3b10db9d8c3cc7a5edc4" } # spiceai
```

Comment:
This needs to be updated once the following PR is merged
- https://github.com/datafusion-contrib/datafusion-table-providers/pull/455